assets/js/main.js

```diff
@@ -0,0 +1,26 @@
+// 3 strings into 1 line
+
+let userName = prompt('User\'s Name');
+let userAge = prompt('User\'s Age');
+let userCity = prompt('User\'s City');
+
+let userInfo = `${userName} ${userAge} ${userCity}`;
+alert(userInfo);
+
+// Decompose number. It's my solution
+
+let someNumber = prompt('Enter the number');
+someNumber = String(someNumber);
+let result = someNumber.split('').join(' ');
```

Comment:
Do you understand how it works? 
Nice solution)

Reply:
Yeah, it was pretty obvious for me...to my surprise xD